R/target_market_share.R

```diff
@@ -420,18 +420,18 @@ aggregate_by_name_abcd <- function(data) {
 format_output_dataframe <- function(data) {
   data <- data %>%
     pivot_wider2(
-      names_from = .data$scenario,
+      names_from = "scenario",
       values_from = c(
-        .data$weighted_production_target,
-        .data$weighted_technology_share_target
+        "weighted_production_target",
+        "weighted_technology_share_target"
       )
     )
 
   data <- data %>%
     rename(
-      weighted_production_projected = .data$weighted_production,
-      weighted_technology_share_projected = .data$weighted_technology_share,
-      sector = .data$sector_abcd
+      weighted_production_projected = "weighted_production",
+      weighted_technology_share_projected = "weighted_technology_share",
+      sector = "sector_abcd"
```

Comment:
why literal strings here instead of using `all_of()` as above, e.g. R/join_abcd_scenario.R#L54?

Reply:
Because I realised part way through that plain strings are much simpler.

Reply:
Thanks for the explanation 🙏🏻